dist/overflow-color.umd.js

```diff
@@ -57,14 +45,18 @@
   var checkScroll = function checkScroll() {
     lastScrollY = window.scrollY;
     if (!ticking && (topColor || bottomColor)) {
-      requestAnimFrame(function () {
-        var scrollHeight = document.body.scrollHeight;
-        var innerHeight = window.innerHeight;
-        if (scrollHeight === innerHeight) {
+      var scrollHeight = document.body.scrollHeight;
+      var innerHeight = window.innerHeight;
+
+      requestAnimFrame(() => {
+        if (lastScrollY > 0) {
```

Comment:
Here we just separate the ternary operator in two conditions, because the ternary operator didn't cover one more condition when `lastScrollY` sets to 0. And it happening on the page without scroll. So in order to get rid of the flickers while scrolling up and down too fast, it works well with this approach.